modules/search/instant-search/components/search-results.jsx

```diff
@@ -3,23 +3,90 @@
 /**
  * External dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
+import { __, _n, sprintf } from '@wordpress/i18n';
 import { h, Component } from 'preact';
 
 /**
  * Internal dependencies
  */
-import SearchResult from './search-result';
+import SearchResultMinimal from './search-result-minimal';
+import { hasFilter } from '../lib/query-string';
+import ScrollButton from './scroll-button';
 
 class SearchResults extends Component {
+	renderResult = ( result, index ) => {
+		switch ( this.props.resultFormat ) {
+			case 'engagement':
+			case 'product':
+			case 'minimal':
+			default:
+				return (
+					<SearchResultMinimal
+						index={ index }
+						locale={ this.props.locale }
+						query={ this.props.query }
+						result={ result }
+					/>
+				);
+		}
+	};
+
 	render() {
-		const { results = [], query } = this.props;
+		const { query } = this.props;
+		const { results = [], total = 0, corrected_query = false } = this.props.response;
+		const hasQuery = query !== '';
+		const hasCorrectedQuery = corrected_query !== false;
+		if ( ! hasQuery && ! hasFilter() ) {
+			return <div className="jetpack-instant-search__search-results" />;
+		}
+		if ( total === 0 ) {
+			return (
+				<div className="jetpack-instant-search__search-results">
+					<div>
+						<h3>{ sprintf( __( 'No Results.', 'jetpack' ), query ) }</h3>
+					</div>
+				</div>
+			);
+		}
+		const num = new Intl.NumberFormat().format( total );
+
+		let headerText = sprintf( _n( '%s result', '%s results', total ), num );
```

Comment:
Do we need to add the textdomain when using `_n` here and below?
https://github.com/WordPress/gutenberg/tree/master/packages/i18n#usage